### PR TITLE
Permutation importance for logistic/linear regressions.

### PR DESCRIPTION
**Pull request by kei51e**
# Description
Permutation importance for logistic/linear regressions.

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/build_lm.R [hunk 1]
```diff
@@ -1,3 +1,27 @@
+calc_permutation_importance_logistic <- function(fit, target, vars, data) {
```
Comment (kei51e) [inline R/build_lm.R:1]: Add comment.

File: R/build_lm.R [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+  })
+  importances <- purrr::flatten_dbl(importances)
+  importances_df <- tibble(term=vars, importance=importances)
+  importances_df
+}
+
+calc_permutation_importance_linear <- function(fit, target, vars, data) {
+  var_list <- as.list(vars)
+  importances <- purrr::map(var_list, function(var) {
+    mmpf::permutationImportance(data, var, target, fit, nperm = 1, # By default, it creates 100 permuted data sets. We do just 1 for performance.
```
Comment (kei51e) [inline R/build_lm.R:17]: Add a description why `predict.fun` is missing here.

File: R/build_lm.R [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+                                # For some reason, default loss.fun, which is mean((x - y)^2) returns NA, even with na.rm=TRUE. Rewrote it with sum() to avoid the issue.
+                                loss.fun = function(x,y){sum((x - y)^2,na.rm = TRUE)/length(x)})
```
Comment (kei51e) [inline R/build_lm.R:19]:
```suggestion
                                loss.fun = function(x,y){sum((x - y)^2, na.rm = TRUE)/length(x)})
```

File: R/build_lm.R (hunk 1 above)
Comment (hideaki) [reply R/build_lm.R:1]: Done.

File: R/build_lm.R (hunk 2 above)
Comment (hideaki) [reply R/build_lm.R:17]: Done.

**Pull request merged**